widget/embedded/src/components/ConfirmWalletsModal/ConfirmWallets.styles.ts

```diff
@@ -115,11 +113,11 @@ export const CollapsibleRoot = styled(Collapsible.Root, {
 
 export const CustomDestinationButton = styled(Button, {
   width: '100%',
-  borderRadius: '$10',
+  borderRadius: '$sm !important',
```

Comment:
Why we need `!important`, it should override the `Button`. Doesn't it?

Reply:
No, it doesn't override the Button styles without important keyword. the Button variants styles overrides this styles.

Reply:
I don't agree with using `!important` because it shows we couldn't handle the situation well and force it using `important`. 
You don't need to fix this right now, we will check it in future.